apps/script/checkEntryPoints.js

```diff
@@ -0,0 +1,223 @@
+var chalk = require('chalk');
+var child_process = require('child_process');
+var path = require('path');
+var fs = require('fs');
+
+/**
+ * Configuration for entry points. Mostly this just silences all the bad entry points
+ * we have right now.
+ */
+const SILENCE_OUT_OF_SITES = [
+  'applab-api',
+  'tutorialExplorer',
+  'makerlab',
+  'pd',
+  'publicKeyCryptography',
+  'brambleHost',
+];
+const SITES_CONFIG = {
+  'studio': {
+    templateRoot: '../dashboard/app/views',
+    templateGlobs: ['**/*.erb', '**/*.haml'],
+    templateExtensions: ['erb', 'haml'],
+    silence: [
```

Comment:
might be worth a comment that (a) we silence these because they are existing violations (b) we shouldnt add new violations and (c) we'd like to fix up our old violations over time.